pandas/tests/indexes/datetimes/test_constructors.py

```diff
@@ -931,7 +931,7 @@ def test_is_(self):
         dti = date_range(start="1/1/2005", end="12/1/2005", freq="M")
         assert dti.is_(dti)
         assert dti.is_(dti.view())
-        assert not dti.is_(dti.copy())
+        assert dti.is_(dti.copy())
```

Comment:
why would this change?

Reply:
I may havd misread the doc string, but the thought was that if `is_` compares to equal for views, it should also do it for copies, because indexes are immutable.

Reply:
no this means they are the same object 

that means that a copy is not actually a copy but the same object

Reply:
> that means that a copy is not actually a copy but the same object

Do you mean `index.copy(deep=False)` is a _new_ object viewing the same data?

Reply:
Given the ambiguity here, would it make sense to rename `is_` to something more precise?  maybe `_views_same_data`?